.github/workflows/ci.yml

```diff
@@ -189,17 +191,34 @@ jobs:
       run: zypper --non-interactive install rustup
 
     - name: Install required packages
-      run: zypper --non-interactive install  python-langtable-data openssl-3 libopenssl-3-devel
+      run: zypper --non-interactive install  python-langtable-data openssl-3 libopenssl-3-devel jq
 
     - name: Install Rust toolchains
       run: rustup toolchain install stable
 
+    - name: Install cargo-binstall
+      uses: taiki-e/install-action@v2
+      with:
+        tool: cargo-binstall
+
+    - name: Install Tarpaulin (for code coverage)
+      run: cargo-binstall --no-confirm cargo-tarpaulin
```

Comment:
so we have third way to install rust stuff? rustup component, cargo install and now cargo-binstall? It start to be quite messy

Reply:
Just to clarify: rustup is for managing the rust toolchain. `cargo install` is for binary crates (building for sources) and cargo-binstall to avoid building for sources.